src/main/resources/data-repository-openapi.yaml

```diff
@@ -276,6 +290,23 @@ paths:
           description: Not found - job id does not exist
 
 
+  '/api/repository/v1/jobs/{id}/result':
+    get:  ## retrieve
+      description: Retrieve a job's result by id
+      operationId: retrieveJobResult
+      tags:
+      - repository
+      parameters:
+      - $ref: '#/parameters/Id'
+      responses: ## successful responses return the type of model specified by the job
+        default:
+          schema:
+            type: object
+        400:
```

Comment:
We need the object return for everything. That is because if the flight fails, the status code will be from the flight and the ErrorModel will be the returned object.